collector/consumer/processor/kindlingformatprocessor/protocol.go

```diff
@@ -74,6 +80,17 @@ func fillTopologyHttpProtocolLabel(g *gauges) {
 	g.targetLabels.AddStringValue(constlabels.StatusCode, strconv.FormatInt(g.Labels.GetIntValue(constlabels.HttpStatusCode), 10))
 }
 
+func fillSpanHttpProtocolLabel(g *gauges) {
+	g.targetLabels.AddStringValue("http.method", g.Labels.GetStringValue(constlabels.HttpMethod))
+	g.targetLabels.AddStringValue("http.endpoint", g.Labels.GetStringValue(constlabels.HttpUrl))
+	g.targetLabels.AddIntValue("http.status_code", g.Labels.GetIntValue(constlabels.HttpStatusCode))
+	// TODO trace_id and trace_type
+	//kv = append(kv, attribute.String("http.trace_id", s.Protocol.HTTP.TraceID))
+	//kv = append(kv, attribute.String("http.trace_type", s.Protocol.HTTP.TraceType))
+	g.targetLabels.AddStringValue("http.request_payload", g.Labels.GetStringValue(constlabels.HttpRequestPayload))
+	g.targetLabels.AddStringValue("http.response_payload", g.Labels.GetStringValue(constlabels.HttpResponsePayload))
```

Comment:
The data model actually requires the fields of `http.request_headers`, `http.request_body`, `http.response_headers`, `http.response_body`, that should be changed.

<img width="843" alt="image" src="https://user-images.githubusercontent.com/46807570/156490364-44ca95a1-6202-41b1-ae12-9621e16693b5.png">